Kinvey/Kinvey/Client.swift

```diff
@@ -267,6 +273,7 @@ open class Client: Credential {
         if let user = keychain.user {
             user.client = self
             activeUser = user
+            clientId = keychain.clientId
             let customUser = user as! U
             completionHandler(.success(customUser))
         } else if let kinveyAuth = sharedKeychain?.kinveyAuth {
```

Comment:
My understanding is that the `client_id` in a shared keychain will still correspond to the active user, so we should treat it exactly the same way. This assumes that 2 apps that want to share a MIC config will also share the same `client_id`.